server/publications/collections/accounts.js

```diff
@@ -45,6 +45,22 @@ Meteor.publish("Accounts", function (userId) {
 });
 
 /**
+ * Single account
+ * @params {String} userId -  id of user to find
+ */
+Meteor.publish("Accounts.single", function (userId) {
```

Comment:
@aaronjudd Should this be `AccountsSingle`? 
Reason for creating this publication: The existing `Accounts` publication doesn't allow doing a `find()` with `userId` passed in. Some Avalara tax setting needs to be set on individual accounts, hence I created this.